services/src/test/java/org/fao/geonet/api/users/UsersApiTest.java

```diff
@@ -85,7 +86,7 @@ public void getUsers() throws Exception {
             .session(this.mockHttpSession)
             .accept(MediaType.parseMediaType("application/json")))
             .andExpect(status().isOk())
-            .andExpect(jsonPath("$", hasSize(5)))
+            .andExpect(jsonPath("$", hasSize(6)))
```

Comment:
Not sure if this change is related to the PR?

Reply:
No, but the test was failing (because of https://github.com/geonetwork/core-geonetwork/blob/main/web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql#L729)